queries.sql

```diff
@@ -15,4 +15,53 @@ SELECT * FROM animals WHERE neutered = 'true';
 -- Find all animals not named Gabumon.
 SELECT * FROM animals WHERE name != 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
-SELECT * FROM animals WHERE weight_kg = 10.4 OR 17.3 OR weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg = 10.4 OR 17.3 OR weight_kg BETWEEN 10.4 AND 17.3;
+-- nside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+-- Inside a transaction:
+-- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
+-- Commit the transaction.
+-- Verify that change was made and persists after commit.
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE name NOT LIKE '%mon';
+ COMMIT;
```

Comment:
- [ ] Please follow the example above to verify that change was made, and also verify that change persist after commit as [requested](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md) for this project, " `Verify that change was made and persists after commit.` ".